packages/block-library/src/cover/save.js

```diff
@@ -86,21 +82,31 @@ export default function save( { attributes } ) {
 		},
 		getPositionClassName( contentPosition )
 	);
-
+	/**
+	 * We default to `black` background to enable the opacity of
+	 * images/videos when we haven't explicitly set any overlay color.
+	 */
+	const usedColor = overlayColor || customOverlayColor || 'black';
 	return (
 		<div { ...useBlockProps.save( { className: classes, style } ) }>
-			{ url && ( gradient || customGradient ) && dimRatio !== 0 && (
+			{ dimRatio !== 0 && (
 				<span
 					aria-hidden="true"
 					className={ classnames(
 						'wp-block-cover__gradient-background',
```

Comment:
We are using this span even for colors now. Should we include a general class for this element e.g: wp-block-cover__background,  and only include the old wp-block-cover__gradient-background on gradients for back-compatibility?
We may also discard the class old class and include migrations or introduce a new class wp-block-cover__color-background that is used for colors keeping wp-block-cover__gradient-background on gradients.